README.md

```diff
@@ -83,20 +83,149 @@ Once the configuration is done and this directory is copied to `dom0`, you must
 make all
 ```
 
-The build process takes quite a while. You will be presented with a dialogue asking how to connect to Tor: you should be able to select the default option and continue. You may wish to increase the scrollback in the dom0 terminal from 1000 (the default) to 100000, to ensure you can review any errors in the verbose output.
+The build process takes quite a while. You will be presented with a dialog asking how to connect to Tor: you should be able to select the default option and continue. You may wish to increase the scrollback in the dom0 terminal from 1000 (the default) to 100000, to ensure you can review any errors in the verbose output.
+
+**NOTE:** Due to [issue #202](https://github.com/freedomofpress/securedrop-workstation/issues/202), the installation may fail with disk quota errors. If this happens, reboot the entire workstation and run `make all` again.
 
 When the installation process completes, a number of new VMs will be available on your machine, all prefixed with `sd-`.
 
-*note* If you've previously installed sd-workstation before January 2018, you may need to run this in dom0 once, by hand, in order to install cleanly:
+#### Using the SecureDrop Client
```

Comment:
Nit: the `####` header should be `###`, to make it a sibling section of "Installation". Similarly, the subsections should have -1 in their header declarations.